### DISCUSSION TITLE
Icon request: icon-www (website)

### DISCUSSION THREAD

**Discussion by girlpunk** [FortAwesome/Font-Awesome#21121] [Icon Requests]
A styalised "www" icon to represent an individual or company's website

Thanks!

**Comment by tagliala** [comment]
IMHO `www` is hard to fit in a little 14x14 px icon.

I'm using `icon-globe` at the to represent a website at the moment

**Comment by tagliala** [comment]
google is also using a globe to represent websites

![screenshot_2013-07-20-17-29-57](https://f.cloud.github.com/assets/556268/830457/1d47c2f6-f152-11e2-9c02-92631fe26c15.png)

**Comment by gaplauche** [comment]
Here's an icon I see used to represent websites:

![](http://whmultimedia.com/wp-content/uploads/2012/09/www_icon.jpg)

I'd like to see something like this added.

**Comment by richardczeiger** [comment]
+1

**Comment by tagliala** [comment]
@gaplauche @richardczeiger  this icon is a request for "www" inside an icon.

I don't know what to do, but if you need a wireframe globe, please +1 #2655

**Comment by seezee** [comment]
+1 Here is what I'd like to see. https://cloud.githubusercontent.com/assets/287836/6373374/e80de34a-bccf-11e4-9ccb-2305fdf74305.png

**Comment by peterppm** [comment]
- 1 on a website icon.

**Comment by StanfordAlan** [comment]
+1 for an icon that's clearly a website instead of a location like the globe could represent. Thanks!

**Comment by peterjacobson** [comment]
+1

**Comment by jamesokbo** [comment]
+1

**Comment by hasan-r0ny** [comment]
+1 For this icon! We need it.

**Comment by Erwane** [comment]
+1 for this

**Comment by danielmorlock** [comment]
+1

**Comment by 78wesley** [comment]
+1

**Comment by sensibleworld** [comment]
This is a quick update to let folks know that we’re refining our icon request workflow behind the scenes, and as part of that we’re going through all open requests and filing them into several new buckets to be addressed.

In this case we’ve added this issue to our new list to be created! We can’t promise any firm timelines right now, but this will help us keep track of it.

We’ll update this issue when the icon is released into the wild 🙂

**Reply by peterppm** [reply]
Thanks for the update.
It's interesting to note this request started in 2013!
Nearly as old as some of my jokes.

**Comment by sensibleworld** [comment]
https://fontawesome.com/icons/globe-www?f=classic&s=solid
